readthedocs/templates/404.html

```diff
@@ -16,6 +16,10 @@
 {% block language-select-form %}{% endblock %}
 
 {% block content %}
+<p>{% blocktrans context "404 page" %}This is a 404 error page. The link you have followed or the URL that you entered did not work. It might be that the documentation has been updated.{% endblocktrans %}</p>
```

Comment:
```suggestion
<p>{% blocktrans context "404 page" %}This is a 404 error page. The link you have followed or the URL that you entered does not exist. It might be that the documentation has been updated, or this page has been removed.{% endblocktrans %}</p>
```

We should be explicit that 404 means doesn't exist, not "did not work".

Reply:
Good change!